.github/workflows/create_sbom_report.yml

```diff
@@ -0,0 +1,119 @@
+name: Create and upload a SBOM to release assets
+# Inherited variables:
+# github.event.client_payload.imageLabel - AzDO image/OS label
+# github.event.client_payload.GHreleaseID - Current release ID
+# github.event.client_payload.imageVersion - AzDO image/OS version "major.minor"
+# 
+# Current SYFT tool issues:
+# macOS (minor): very long cataloging process (https://github.com/anchore/syft/issues/1328), 
+# macOS (major): promt privilegies that blocking process indefinetely (https://github.com/anchore/syft/issues/1367)
+# Windows (major): process fails on some symlinks (https://github.com/anchore/syft/issues/1094)
+on:
+  repository_dispatch:
+    types: [generate-sbom]
+jobs:
+  #Converting image OS variable for the next steps
+  initialize: 
+    strategy:
+      matrix: #Couples AzDO label <-> GHA image spec
+        include:
+          - label: ubuntu22
+            os: ubuntu-22.04
+          - label: ubuntu20
+            os: ubuntu-20.04
+          - label: ubuntu18
+            os: ubuntu-18.04
+          - label: macOS-12
+            os: macos-12
+          - label: macOS-11
+            os: macos-11
+          - label: macOS-10.15
+            os: macos-10.15
+          - label: win22
+            os: windows-2022
+          - label: win19
+            os: windows-2019
+    runs-on: ubuntu-latest
+    steps:
+      - name: Convert image label variable
+        run: if ("${{ github.event.client_payload.imageLabel }}"" -eq "{{ $matrix.label }}"") {echo "current-os=${{ $matrix.os }}" >> $GITHUB_OUTPUT}
+  #Checking image version on available runner
+  version-check:
+    strategy:
+      matrix:
+        include:
+          - os: Windows
+            path: C:\
+          - os: macOS
+            path: /Users/runner/
+          - os: Linux
+            path: /imagegeneration/
+    defaults:
+      run:
+        shell: pwsh
+    needs: initialize
+    runs-on: ${{ needs.initialize.output.current-os }}
```

Comment:
In current implementation you can't use this variable. Syntax `echo "var=value" >>$GITHUB_OUTPUT` sets variable on step level by default and it is not exposed to different jobs.
If you want to consume variable from different job you should define it as job output. See this example: https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs

Reply:
Accepted and corrected.